guides/v2.2/extension-dev-guide/code-generation.md

```diff
@@ -48,8 +45,15 @@ Code generation is required in Magento 2. Generating code assures you of the fol
 	All generated Factories work the same way. After you know how one Factory works, you know how they all work.
 
 
-<h2 id="codegen-om">Object Manager responsibility for code compilation</h2>
+## Object Manager responsibility for code compilation {#codegen-om}
 When code changes as discussed in the preceding section, the Object Manager compiles it.
 
-
 The code compiler creates `generated/metadata/global.php`, which is a PHP serialized map of all constructor definitions mixed with object linking configuration defined in di.xml. `di.xml` is the dependency injection configuration. There is a global `app/etc/di.xml` and there can one defined for every {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %}.
+
+<!-- Link Definitions -->
+[\\Magento\\Customer\\Model\\Resource\\AddressRepository]: {{ site.mage2000url }}/app/code/Magento/Customer/Model/ResourceModel/AddressRepository.php
```

Comment:
Slashes and target code formatting - see previous comments.

Reply:
Fixed